ui/partials/perf/alertsctrl.html

```diff
@@ -0,0 +1,99 @@
+<div class="container-fluid">
+  <div class="alert alert-danger" role="alert">
+    Perfherder alerts are experimental and not to be trusted (yet)
+  </div>
+  <hr/>
+  <div class="panel panel-default" ng-repeat="alertSummary in alertSummaries">
+    <div class="panel-heading">
+      <a href="#/alerts?id={{alertSummary.id}}" class="alert-title">{{alertSummary.title}} <span class="fa fa-external-link icon-superscript"/></a>
+      <ul class="list-unstyled alert-metadata">
+        <li>{{alertSummary.platforms.join(' / ')}}</li>
+        <li>{{alertSummary.resultSetMetadata.dateStr}}</li>
+        <li ng-if="alertSummary.pushURL">
+          <a href="{{alertSummary.pushURL}}">{{alertSummary.resultSetMetadata.revision}}…{{alertSummary.prevResultSetMetadata.revision}}</a>
+        </li>
+        <li ng-if="!alertSummary.pushURL">
+          <!-- This shouldn't happen in production, only in development -->
+          <span class="text-danger">Unknown revision(s)</span>
+        </li>
+      </ul>
+    </div>
+    <div class="panel-body">
+      <h4>Alerts</h4>
+      <table class="table table-striped compare-table">
+        <tr>
+          <!-- Manually specify table widths because it's just easier this way -->
+          <th class="test-title">
+            <span style=" word-wrap: break-word;">
+              Test
+            </span>
+          </th>
+          <th style="width: 140px;">Previous</th>
+          <th style="width: 30px;"><!-- less than / greater than --></th>
+          <th style="width: 140px;">New</th>
+          <th style="width: 80px;">Delta</th>
+          <th style="width: 120px"><!-- Graphical difference --></th>
+          <th style="width: 100px;">Confidence</th>
+        </tr>
+        <tr ng-repeat="alert in alertSummary.alerts">
+          <td class="test-title">
+            {{alert.title}}&nbsp;&nbsp;
+            <span class="result-links">
+              <a href="#/graphs?timerange={{alertSummary.resultSetMetadata.timeRange}}&series=[{{alertSummary.repository}},{{alert.signature_hash}},1]&highlightedRevisions={{alertSummary.resultSetMetadata.revision}}">graph</a>
```

Comment:
Not always, it could be longer in the case of an old alert.

See: https://github.com/mozilla/treeherder/pull/1149/files#diff-d5c3abf248b638a13014d6259abba31aR88